conf/templates/my-account-forgot-password-success.json

```diff
@@ -0,0 +1,5 @@
+{
+	"header" : { "$ref" : "common/header.json" },
+	"footer" : { "$ref" : "common/footer.json" },
+    "meta" : { "$ref" : "common/meta.json" }
+}
```

Comment:
We don't need JSON files as input for the handlebars templates, we have the app input for that. If you keep it here just as a reminder of creating it in the Theme once you import it there, then let's remember to remove it afterwards :)